internal/security/setup/option/constant.go

```diff
@@ -31,6 +32,8 @@ const (
 	envXdgRuntimeDir              = "XDG_RUNTIME_DIR"
 	defaultXdgRuntimeDir          = "/tmp"
 	pkiInitBaseDir                = "/edgex/security-secrets-setup"
+	envPkiCache                   = "PKI_CACHE"
+	defaultPkiCacheDir            = "/etc/edgex/pki"
 	tmpfsRunDir                   = "/run"
 	tlsSecretFileName             = "server.key"
```

Comment:
Would this be a problem if we decide to run the pki-init native in the future ( E.g, running the binary in the windows machine) ?

Reply:
> Would this be a problem if we decide to run the pki-init native in the future

@tingyuz Please elaborate on what you mean by "this".  Path names obviously need to change on Windows. The PKI_CACHE env var will be replaced in a future PR with configuration.toml.  On Windows platforms, enabling BitLocker would help protect TLS assets when the system is powered-off.

Reply:
Hard-coded path is not flexible. It's good if we have plan to move it into configuration.toml or other places so that it becomes configurable. I think we are good here.